beacon-chain/execution/service.go

```diff
@@ -413,29 +413,30 @@ func (s *Service) batchRequestHeaders(startBlock, endBlock uint64) ([]*types.Hea
 	requestRange := (endBlock - startBlock) + 1
 	elems := make([]gethRPC.BatchElem, 0, requestRange)
 	headers := make([]*types.HeaderInfo, 0, requestRange)
-	errs := make([]error, 0, requestRange)
+	errs := make([]*error, 0, requestRange)
```

Comment:
Same suggestion here.

```suggestion
	// errs must be a slice of pointers to prevent errors being overwritten. See PR#11960 for more context.
	errs := make([]*error, 0, requestRange)
```

Reply:
See comment above